test/integration/ctl/ctl_create_cr_test.go

```diff
@@ -171,15 +195,20 @@ func TestCtlCreateCR(t *testing.T) {
 				}
 			}
 
+			// Try to set Ready Condition if needed, otherwise the test just times out
+			if test.fetchCert {
+				go setCRReadyCondition(t, cmCl, test.inputArgs[0], test.inputNamespace)
+			}
 			// Create CR
 			err = opts.Run(test.inputArgs)
 			if err != nil {
 				// TODO: Maybe it is desirable to make the test more fine grained, i.e. specify which error is expected,
 				// to know where exactly things should fail and then check the correctness of the parts that shouldn't have failed
```

Comment:
This wasn't added here but.. yes, definitely 😄  especially as this output is user-facing in their CLI, so it's good to have the actual text encoded here so we know when we change it (either by accident or intentionally). It also helps us avoid doing something like attempting to print a structure or a pointer or something to the CLI, which is really confusing for people 😅